applications/NXmx.nxdl.xml

```diff
@@ -897,6 +935,13 @@
                     </field>
                 </group>
             </group>
+
+            <group type="NXmonitor" minOccurs="0">
```

Comment:
This monitor is optional and has no recommended name or specific use case. Is it really helpful to make such a vague suggestion?

Reply:
This is optional because the user must select one of the 5 options.  It it's vague because it's a placeholder for future use.